apps/vscode/test_server.py

```
@@ -295,12 +347,168 @@ def run_tests(base_dir: Path) -> bool:
+    manage_env = os.environ.copy()
+    manage_env.setdefault("DJANGO_SETTINGS_MODULE", "config.settings")
+    create_admin = [
+        sys.executable,
+        str(base_dir / "manage.py"),
+        "create_docs_admin",
+        "--confirm",
+        "--password",
+        admin_password,
+    ]
+    print(f"{PREFIX} Ensuring docs admin user exists before screenshots.")
+    if subprocess.run(create_admin, cwd=base_dir, env=manage_env).returncode != 0:
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)